portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/EmptyState.testcase

```diff
@@ -185,6 +185,226 @@ definition {
 		}
 	}
 
+	@description = "Feature ID: LRAC-10698 | Automation ID: LRAC-11119 | Test Summary: Empty state of no assets with data source and property"
```

Comment:
Hey @joshitagaki, I think this test is from this story [LRAC-10405](https://issues.liferay.com/browse/LRAC-10405)

`Feature ID: LRAC-10405`